cmd/src/testdata/TestSearchStream/Text.golden

```diff
@@ -1,8 +1,8 @@
 [38;5;23morg/repo[0m › [38;5;69mpath/to/file[0m[38;5;2m (2 matches)[0m
 [38;5;239m--------------------------------------------------------------------------------
-[0m  [38;5;69m     5[0m[38;5;239m |  [0m[38;5;0m[48;5;11mfoo[0m bar [38;5;0m[48;5;11mfoo[0m
-
-[38;5;2msourcegraph/sourcegraph[0m[38;5;239m ([0m[38;5;23mhttp://127.0.0.1:55128/sourcegraph/sourcegraph[0m[38;5;239m)[0m[38;5;2m (1 match)[0m
+[0m  [38;5;69m     5[0m[38;5;239m |  [0m[38;5;0m[48;5;11mfoo[0m bar [38;5;0m[48;5;11mfoo[0m[38;5;239m
+  ------------------------------------------------------------------------------
```

Comment:
There is a minor cosmetic change, that we will put a `----` line below the last result, and is a consequence of keeping the loop logic simple